include/roboptim/core/detail/utility.hh

```diff
@@ -34,8 +34,8 @@
 
 # include <vector>
 
-# include <Eigen/Core>
-# include <Eigen/StdVector>
+# include <eigen3/Eigen/Core>
```

Comment:
On my 24.04, `/usr/share/eigen3/cmake/Eigen3Targets.cmake` contains:
```cmake
set_target_properties(Eigen3::Eigen PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include/eigen3"
)
```

Therefore, if you link your libraries to the `Eigen3::Eigen` CMake export, you don't need to add this `eigen3/` to your c++ `#include <…>`

Reply:
Fixed with 2d9847d

Reply:
The issue was not about `Eigen/Stdvector` in particular, but with the `<Eigen/xxx>` -> `<eigen3/Eigen/xxx>` change : The correct way of including Eigen things is the former, so the changes in this PR on this topic are going in the wrong direction

Reply:
Sorry, for some reason your comment on my commit hit my brain as a suggestion...
... commit 8c353f83 should provide what is expected.